thatkitebot/cogs/welcomecog.py

```diff
@@ -0,0 +1,135 @@
+# ------------------------------------------------------------------------------
+#  MIT License
+#
+#  Copyright (c) 2019-2021 dimonDDL
```

Comment:
uh sir you are now claiming ownership of the entire bot? Because this cog has not existed in 2019

Reply:
Mm. I claim ownership of the file so let me update the date.